taas/__init__.py

```diff
@@ -141,15 +144,36 @@ def normalise_sheet(raw, mapping):
 
         Raw is the data as produced from read_sheet_csv().
         Mapping is a dict of cooked->raw pairs (eg: 'label' : 'Preferred Term')
+
+        Fields containing a . in their name will be turned into maps. Eg:
+        `label.en`, `label.es` and `label.fr` become three keys in the `label` map.
+        Only one level of embedding is supported.
```

Comment:
Aha! Perhaps something like "as of date|commit we support one level of embedding; this could be extended in future"?